### PR TITLE
Make Tomasz and Karsten owners.

### PR DESCRIPTION
**Pull request by jeschkies**

### CODE REVIEW

**Comment by jeschkies** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal

File: owners.json [hunk 1]
```diff
@@ -18,5 +18,7 @@
   "urbanserj": {"slack": "urbanserj", "jira": "sergeyurbanovich"},
   "vinodkone": {"slack": "vinodkone", "jira": "vinod"},
   "Poltergeist": {"slack": "philipph", "jira": "philipph"},
-  "pierrebeitz": {"slack": "pierre", "jira": "pierrebeitz"}
+  "pierrebeitz": {"slack": "pierre", "jira": "pierrebeitz"},
+  "janisz": {"slack": "janisz", "jira": "tomaszjaniszewski"},
```
Comment (janisz) [inline owners.json:22]:
```suggestion
```

I prefer to not be an owner. IMO the only owner should be @orsenthil. Most of our problems are result of previous overrides and merges without tests or lack of tests.

Comment (jeschkies) [reply owners.json:22]: Ownership will make you Github PR approval count for Mergebot as far as I know.

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal

**Pull request merged**